#11028 ensure status 9 advisers are not filtered out

kiggundu opened the pull request.

jordanlg27 commented:
https://maps.tpondemand.com/entity/11028

mrdgoodman commented:
Looks good. Approved.

masbuild commented:
<!--
  0 Errors
  2 Warnings: There is no Targetprocess tick...
  0 Messages
  1 Markdown
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There is no Targetprocess ticket associated to this PR. Please include the link to the corresponding task (TP-XXXX). If no ticket matches these changes consider creating one and linking it here.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There are code changes, but no corresponding tests. Please include tests if this PR introduces any modifications in behavior. If this is a refactor PR, please ignore the message.Refer to our mas standards guides about opening a PR<a href="link-to-the-mas-standards-about-opening-a-pr">here</a></td>
    </tr>
  </tbody>
</table>

At Money Advice Service, we use Danger gem, to ensure that our commits and pull requests follow [standards](https://github.com/moneyadviceservice/mas-standards/blob/master/Dangerfile). If you think one of the points here has been raised in error (or shouldn't be in our standards), open an issue or pull request on [our Danger config](https://github.com/moneyadviceservice/mas-standards).

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

kiggundu commented:
I rebased the branch to root off the current production version, updated the tests and re-wrote commit history to ensure commit id's are the expected TP-XXXX format as per the checks above
